content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Introducing MOSN: An Alternative Data Plane"
+subtitle: "A Cloud Native Proxy for Edge or Service Mesh"
+description: "An alternative sidecar proxy for Istio."
+publishdate: 2020-07-17
+attribution: "Wang Fakang(mosn.io)"
+keywords: [mosn,sidecar,proxy]
+---
+
```

Comment:
Start with an overview like "MOSN (the Modular Open Smart Network) is a network proxy server written in Go. It was built at ___ for ___ /sidecar/use case, and over time, we've added ___ features.  This new ___ thing means we are now able to use MOSN as the network proxy for Istio."

(Other note: you need a "why we built our own" section somewhere.)

Reply:
> Start with an overview like "MOSN (the Modular Open Smart Network) is a network proxy server written in Go. It was built at ___ for ___ /sidecar/use case, and over time, we've added ___ features. This new ___ thing means we are now able to use MOSN as the network proxy for Istio."
> 
> (Other note: you need a "why we built our own" section somewhere.)

Done.

Reply:
The ___ s were for you to fill in the blanks, not to go in as-is :)

It was built at Ant Financial for (whatever use case)... etc.

Reply:
I got it, it has been updated. Thank you.

Reply:
light copy editing:

[MOSN](https://github.com/mosn/mosn) (the Modular Open Smart Network) is a network proxy server written in GoLang. It was built at [Ant Group](https://en.wikipedia.org/wiki/Ant_Financial) as a sidecar/API Gateway/cloud-native Ingress/Layer 4 or Layer 7 load balancer. Over time, we've added extra features, like a multi-protocol framework, multi-process plug-in mechanism, a DSL, and support for the [xDS APIs](https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol). Supporting xDS means we are now able to use MOSN as the network proxy for Istio.